src/models/location.server.model.js

```diff
@@ -3,9 +3,17 @@ const Schema = mongoose.Schema
 const mongoDB = require('../config/mongoose.collections.json')
 
 const locationSchema = new Schema({
-  name: {type: String, index: 1, required: true},
+  name: {type: String, index: 1, required: true, unique: true},
   description: {type: String},
   urlLocation: {type: String},
+  address: {
```

Comment:
@Ivan-Marquez I have a question here. Do I need to write address or addresses? I just have one address